packages/react-reconciler/src/ReactFiberReconciler.new.js

```diff
@@ -153,12 +152,11 @@ function findHostInstance(component: Object): PublicInstance | null {
   const fiber = getInstance(component);
   if (fiber === undefined) {
     if (typeof component.render === 'function') {
-      invariant(false, 'Unable to find node on an unmounted component.');
+      throw new Error('Unable to find node on an unmounted component.');
```

Comment:
Can omit these changes? I don't mind the style fix but seems like it's best done separately